edudoc/delegate-handbook/delegate-handbook.md

```diff
@@ -562,73 +562,73 @@ Above all, the competitor limit must not be exceeded for any reason. Competitor
-  - Inspection time, including penalties
-  - How to start the timer properly
-    - Hand position, particularly using fingers to start timer
-    - Waiting for the green light on the timer
-  - How to stop the timer properly
-    - Flat hands; no karate chops
-  - How to sign for attempts
+Typically, a Delegate is best suited for this task because of their familiarity with the Regulations. In cases where the Delegate doesn't speak the local language, it may be best to choose an organizer or trusted competitor to give the new competitor tutorial. If there are a large number of newcomers, it may be best to have two separate tutorials so that everyone can both see and hear the tutorial more easily.
+
+The following points are an outline of a new competitor tutorial for a WCA competition. Depending on the specific events of the competition, it may be a good idea to have a secondary judging tutorial before an event that has specific judging instructions. For example, 3x3x3 Blindfolded judging does not have the judge time inspection and has them hold a sight blocker, while Clock allows competitors to manipulate pins during inspection, requires judges to check both sides of the puzzle, and does not have +2s. Modify the tutorial as necessary to fit the needs of your competition.
+
+- How to read a nametag, or otherwise determine the competitor's competing, judging, scrambling, and/or running groups.
+- What to do when the competitor's group is called.
+- How to compete.
+  - Telling the judge they are ready.
+  - Inspection time, including penalties.
+  - How to start the timer properly.
+    - Hand position, particularly using fingers to start the timer.
+    - Waiting for the green light on the timer.
+  - How to stop the timer properly.
+    - Flat hands; no "karate chops".
+  - How to sign for attempts.
     - Check that the time matches the timer and is written legibly. Times which are not written legibly will be interpreted in the worst way possible. Competitors can ask their judge to re-write times if they are not easily readable.
-  - How to call over a Delegate
-- How to judge
-  - Asking the competitor if they are ready and waiting for the competitor’s ready signal
-  - Judging inspection, including lifting the cube cover, operating the stopwatch, and making callouts
-    - Emphasize that only "8 seconds" and "12 seconds" are called out, not 15 or 17 seconds
-    - Stopping the stopwatch when the competitor starts their solve, so that the actual time of the inspection is measured
+  - How to call over a Delegate.
+- How to judge.
+  - Asking the competitor if they are ready and waiting for the competitor's ready signal.
+  - Judging inspection, including lifting the cube cover, operating the stopwatch, and making callouts.
+    - Emphasize that only "8 seconds" and "12 seconds" are called out, not 15 or 17 seconds.
+    - Stopping the stopwatch when the competitor starts their solve, so that the actual time of the inspection is measured.
```

Comment:
If you want to match stylistically with the previous bullet, you could change it to start with "Emphasize that they should only stop the stopwatch....".  Also, remove the comma.